Library/Formula/graphene.rb

```diff
@@ -3,6 +3,7 @@ class Graphene < Formula
   homepage "https://ebassi.github.io/graphene/"
   url "https://download.gnome.org/sources/graphene/1.2/graphene-1.2.6.tar.xz"
   sha256 "987a83be0b9e634805d8bb949d11422a42e627fd40f78c68c3b1ac7b3243cda2"
+  revision 1
```

Comment:
Why the `revision` here? Unless it adds functionality that people expect to be there already, I'm fine with not bumping the revision & letting people who want the new feature to just do `brew reinstall graphene`.